docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml

```diff
@@ -1979,6 +1979,131 @@
     </sect3>
   </sect2>
 
+  <sect2 id="GS-LZ-Scopes">
+    <title>Landing Zone Security</title>
+
+    <para>You can set additional security options on Landing Zone(s). Feature
+    level security allows you to set permissions on access to your Landing
+    Zones and what users or groups can do there. Landing Zone Scope Security
+    allows you to set permissions on sub-folders in a Landing zone. This
+    provides a means to grant and deny users permission to areas within a
+    Landing Zone.</para>
+
+    <sect3 id="GS_LandingZoneFeatureAuth">
+      <title>Landing Zone Feature Authorization</title>
+
+      <para>This lists the HPCC Systems Landing Zone using Feature Level
+      Authorization:</para>
+
+      <para><informaltable>
+          <tgroup cols="2">
+            <tbody>
+              <row>
+                <entry>List/search Dropzone files</entry>
+
+                <entry>FileSprayAccess - SecAccess_Read</entry>
+              </row>
+
+              <row>
+                <entry>Spray a file from a Dropzone</entry>
+
+                <entry>FileSprayAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Despray a file to a Dropzone</entry>
+
+                <entry>FileDesprayAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Read the content of a Dropzone file</entry>
+
+                <entry>FileIOAccess - SecAccess_Read</entry>
+              </row>
+
+              <row>
+                <entry>Write the content of a Dropzone file</entry>
+
+                <entry>FileIOAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Upload a file to a Dropzone using ECLWatch:</entry>
+
+                <entry>FileUploadAccess - SecAccess_Full</entry>
+              </row>
+
+              <row>
+                <entry>Download a file from a Dropzone using ECLWatch</entry>
+
+                <entry>FileSprayAccess - SecAccess_Full</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>To enable access to a feature, set the permission
+      accordingly.</para>
+
+      <para>This may be sufficient level security in some cases, however,
+      additional restrictions may be needed to secure certain files, from
+      certain users or groups. You can use Landing Zone File Scope security to
+      accomplish this. .</para>
+    </sect3>
+
+    <sect3 id="LZ_FileScopes">
+      <title>Landing Zone File Scopes</title>
+
+      <para>File Scope Level Authorization provides a means to secure access
+      to folders within a Landing Zone.</para>
+
+      <para>An HPCC Administrator can define one or more Landing Zone scopes
```

Comment:
Can a folder have more than one scope? "An HPCC Administrator can define one or more Landing Zone scopes for each folder..." implies that you can. But, later you say each folder is one scope and each scope is one folder.

Reply:
This sentence is still confusing. The following sentence is a contradiction.